responsibleai/responsibleai/managers/counterfactual_manager.py

```diff
@@ -318,18 +319,18 @@ def load_result(self, data_directory_path):
         else:
             self.counterfactual_obj = None
 
-        result_path = (data_directory_path /
-                       (CounterfactualConfig.HAS_COMPUTATION_FAILED + '.json'))
+        result_path = data_directory_path / \
```

Comment:
I like the slashes much less than grouping via parentheses.  I think we are in favor of parens over slashes in general?  Did you have to make this change due to linting or just personal preference?

Reply:
Well, initially I didn't make any changes. Then flake complained, of course, due to line length, and I fixed those with the most straightforward solution. I can certainly put the parentheses pack, no problem 🙂